democracy/tests/test_hearing.py

```diff
@@ -325,18 +326,26 @@ def test_hearing_copy(default_hearing, random_label):
     assert not new_hearing.sections.filter(type__identifier=InitialSectionType.CLOSURE_INFO).exists()
 
 
+@pytest.mark.parametrize('client, expected', [
+    ('api_client', False),
+    ('jane_doe_api_client', False),
+    ('admin_api_client', True)
+])
 @pytest.mark.django_db
-def test_hearing_open_at_filtering(api_client, default_hearing):
+def test_hearing_open_at_filtering(api_client, default_hearing, request, client, expected):
+    api_client = request.getfuncargvalue(client)
```

Comment:
You're reassigning the param here -- maybe remove `api_client` from the param list?